internal/statestore/redis.go

```diff
@@ -440,7 +440,7 @@ func (rb *redisBackend) FilterTickets(ctx context.Context, filters []*pb.Filter,
 }
 
 func idsToPages(ids []string, pageSize int) [][]interface{} {
-	result := make([][]interface{}, 0, len(ids)/pageSize+1)
+	result := make([][]interface{}, 0, len(ids)/(pageSize+1))
```

Comment:
The previous logic was correct:
Eg, if page size is 10, and there are 15 ids:
(15/10) + 1 = 2

While 15/(10+1) = 1

We are expecting to return two pages, so 2 is the correct answer.  This logic maybe over allocates by 1, but the maximum capacity is roughly known.  Logic works either way, as append will create a new backing array if going over capacity.

Reply:
It's an error with the calling code if it tries to do page size = 0.  Maybe there should be a panic with a better error message, but failing on pagesize=0 seems right to me.

Reply:
Ah - This was originally my hack to work past 0 page size. You are right - this is not needed - and I think failing due to page size 0 here is a code bug - so it is OK to panic. Reverted!